packages/tools/pie-css/css/components.css

```
@@ -0,0 +1,49 @@
+/* This stylesheet assigns a proper display setting to the Web components
+top DOM elements, otherwise the default `inline` will be assigned, leading to
+inconsistent bounding box size and potential layout issues */
```

```suggestion
/* This stylesheet assigns a proper display setting to the host element of the web components, otherwise the default `inline` will be assigned, leading to inconsistent bounding box size and potential layout issues */
```
nit: this will make it a bit clearer that we are referring to the host element tag ex; `<pie-button>`